### ISSUE TITLE
Repeater field html and textarea not displaying properly

### ISSUE THREAD

**Issue by Ed-V** [bolt/bolt#7574]
When using html or text area with repeater field the content stored gets messed up and shows up improperly when pulled.

 

Details
-------

 - Relevant Bolt Version: [ 3.2 | 3.3 | 3.4 | **3.5** | master ]
 - Install type: [ Zip/tarball install | **Composer install** | GitHub checkout ]
 - PHP version: [ 5.5 | 5.6 | **7.0** | 7.1 | 7.2 ]
 - Used web server: [ **Apache** | Nginx ] version [ version ]
 - For UX/UI issues: [ browser name and version ]

Reproduction
------------

If you're filing a bug, please describe how to reproduce it. Include as much
relevant information as possible, such as:

### Bug summary

Using the pulling textarea or html content gets messed up and actually causes the browser to output the actual html. This doesn't occur when using the regular non repeater version of html or textarea

### Specifics

Checked in firefox and chrome.
Using bolt 3.5.5
Checked database the normal fields and the repeated fields store the data identically.

### Steps to reproduce

Create a repeater field with either html or textarea. For example in my config file it looks like this

```
pages:
    name: Pages
    singular_name: Page
    fields:
        title:
            type: text
            class: large
            group: content
        slug:
            type: slug
            uses: title
        image:
            label: Header Image
            type: image
            placeholder: https://source.unsplash.com/1280x768/?business,workplace/__random__
        teaser:
            type: html
            height: 150px
        template:
            type: templateselect
            filter: '*.twig'
        imgtextblocks:
            type: repeater
            group: Image Text Blocks
            fields:
                  content:
                      type: html
                      height: 300px
                  image:
                      type: image
                  imgright:
                      label: Image Right
                      type: checkbox
    taxonomy: [ groups ]
    recordsperpage: 20
```

Everything functions as expected in the editor but when pulling content using something like {{ loopvar.content }} it includes the quotes as well messing up how the browser interperts the content. See images for better example.

### Expected result

Typing `Hello` in the html editor should display `Hello` on the webpage when using {{ loopvar.content }}

### Actual result

In the webpage it displays 

`<p>Hello</p>`

The resulting html code is

`&lt;p&gt;Hello&lt;/p&gt;`

The below image shows the source to illustrate that it looks fine in the editor and that there aren't any other random tags there but doesn't properly fetch content.
![editor](https://user-images.githubusercontent.com/10635618/43236153-d32c72e6-9040-11e8-8bc3-a94aa59a8951.PNG)
![webpage](https://user-images.githubusercontent.com/10635618/43236161-dabb6b2a-9040-11e8-9418-db451b7a495e.PNG)
![inspector](https://user-images.githubusercontent.com/10635618/43236782-1d544c74-9044-11e8-87c5-5ac0de13b68e.PNG)

Using the regular non repeater fields the dump looks like this, hopefully this will help.

![regular dump](https://user-images.githubusercontent.com/10635618/43238166-43d9c0c6-904a-11e8-9845-816bbf9e5fca.PNG)

**Comment by Ed-V** [comment]
Read around and fixed it by using `|raw()` which is apparently hackish. Not sure if this was intended or not.

**Comment by rossriley** [comment]
@Ed-V it is a bit hackish, we've got a proper fix coming in 3.6 which means that using raw won't be necessary any longer, but for now it is the correct way.

**Comment by bobdenotter** [comment]
Closing this one! Not because it's a bad issue, but because it's resolved in 3.6, and i'd like to clean up the list a little. 😇
